<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Flight_Of_The_Clones</class>
 <widget class="QMainWindow" name="Flight_Of_The_Clones">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1076</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(62, 130, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="Btn_play">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>370</y>
      <width>251</width>
      <height>121</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 26pt &quot;Berlin Sans FB&quot;;
background-color: rgb(204, 204, 204);</string>
    </property>
    <property name="text">
     <string>Play</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_best">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>10</y>
      <width>201</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 24pt &quot;Berlin Sans FB&quot;;
background-color: rgb(204, 204, 204);</string>
    </property>
    <property name="text">
     <string>Best players</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Authors">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>211</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Berlin Sans FB</family>
      <pointsize>20</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 20pt &quot;Berlin Sans FB&quot;;
background-color: rgb(204, 204, 204);</string>
    </property>
    <property name="text">
     <string>About developers</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1076</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
